pkg/build/controller/policy/serial.go

```diff
@@ -20,7 +20,7 @@ type SerialPolicy struct {
 func (s *SerialPolicy) IsRunnable(build *buildapi.Build) (bool, error) {
 	bcName := buildutil.ConfigNameForBuild(build)
 	if len(bcName) == 0 {
-		return false, NewNoBuildConfigLabelError(build)
+		return true, nil
```

Comment:
(I think it might be OK, as there is no BC associated to build which means those builds should have parallel policy)